app/models/ontology/class_methods_and_scopes.rb

```diff
@@ -76,7 +76,7 @@ module Ontology::ClassMethodsAndScopes
 
   module ClassMethods
     def find_with_locid(locid, iri = nil)
```

Comment:
Isn't this exactly what you do in the parent class? This method seems redundant.